brainstormforce/ultimate-addons-for-gutenberg
# Improvement: Marketing button block default styling changes

### Description
Updated Marketing button default design

### Screenshots


### Test Cases 1 

- Switch to master branch
- Drag and drop marketing button with different stylings and options
- Save page and switch to marketing-btn-default-bg branch
- Check editor it is working properly or not for backward compatibility

### Test Cases 2 

- Switch to marketing-btn-default-bg branch
- Drag and drop marketing button
- Check default wp button styling applying or not like border, bg color and text color

### Types of changes
breaking change which fixes an issue

### How has this been tested?


### Checklist:
